cmd/private-prow-configs-mirror/main.go

```diff
@@ -290,73 +290,55 @@ func injectPrivateBugzillaPlugin(bugzillaPlugins plugins.Bugzilla, orgRepos orgR
 }
 
 func injectPrivatePlugins(plugins map[string][]string, orgRepos orgReposWithOfficialImages) {
-	logrus.Info("Processing...")
-
-	var allPrivate []string
-
 	privateReposWithValues := make(map[string][]string)
```

Comment:
There's no reason to talk about "values" here, we are dealing with "plugins" here, so let's use that term everywhere (`privateRepoPlugins` instead of `privateReposWithValues`, `commonPlugins` instead of `commonValuesSet`, `getCommonPlugins` instead of `getCommonValues` etc).